components/command/command-handler/src/main/java/uk/gov/justice/services/core/aggregate/AggregateService.java

```diff
@@ -26,38 +32,50 @@
     Logger logger;
 
     @Inject
+    SnapshotService snapshotService;
+
+    @Inject
     JsonObjectToObjectConverter jsonObjectToObjectConverter;
 
     private ConcurrentHashMap<String, Class<?>> eventMap = new ConcurrentHashMap<>();
 
+    private ObjectInputStreamStrategy objectInputStreamStrategy = new DefaultObjectInputStreamStrategy();
```

Comment:
If this is only set to a different implementation for testing, then the setObjectInputStreamStrategy method needs to package local